docs/interface_lang_management.md

```diff
@@ -0,0 +1,62 @@
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Language management interface
+
+Eclipse OpenJ9 provides MXBean extensions to the standard `java.lang.management` API, which can be used to monitor and manage the Java@trade; virtual machine.
+These extensions provide access to information about the state of the OpenJ9 VM and the environment in which it is running. The following
+tables list the MXBeans by package and describe the monitoring or management capability:
+
+
+| `com.ibm.lang.management` MXBean  | Description                                                                                                  |
+|---------------------------|--------------------------------------------------------------------------------------------------------------|
+| `GarbageCollectorMXBean`    | Discovers Garbage Collection (GC) operations (collection times, compactions, heap memory usage, and freed memory). |
+| `JvmCpuMonitorMXBean`       | Discovers CPU consumption by category (GC, JIT, or other threads).                                             |
+| `MemoryMXBean`              | Discovers memory usage (minimum and maximum heap sizes, and shared class cache sizes).               |
+| `MemoryPoolMXBean`          | Discovers memory pool usage for specific GC policies.                                                         |
+| `OperatingSystemMXBean`     | Discovers information about the operating system (memory, CPU capacity/utilization).                         |
+| `RuntimeMXBean`             | Discovers information about the runtime environment (CPU load, Java process ID, and VM state)                |
+| `ThreadMXBean`              | Discovers information about native thread IDs.                                                                |
+| `UnixOperatingSystemMXBean` | Discovers information for Unix operating systems (memory, file descriptors, processors, processor usage, and hardware)|
+
+| `com.ibm.virtualization.management` MXBean | Description                                                                                                  |
+|---------------------------|--------------------------------------------------------------------------------------------------------------|
+| `GuestOSMXBean`             | Discovers CPU and memory statistics of a virtual machine or logical partition as seen by the Hypervisor.       |
+| `HypervisorMXBean`          | Discovers whether the operating system is running on a hypervisor and provides information about the hypervisor.|
+
+
+| `openj9.lang.management` MXBean | Description                                                                                                  |
```

Comment:
Can the header be forced onto two lines like the others above?

Reply:
Tables are quite restrictive in markdown. I lifted the package names out as headers to the tables - hopefully make it look a bit better. Staging now.